packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -724,23 +752,19 @@ class _StretchingOverscrollIndicatorState extends State<StretchingOverscrollIndi
           final double stretch = _stretchController.value;
           double x = 1.0;
           double y = 1.0;
-          final AlignmentDirectional alignment;
+          late final AlignmentDirectional alignment;
```

Comment:
I think we can omit the `late` keyword. Seems like the compiler is able to tell that this variable will definitely be assigned.

Note: I tried this on DartPad and it worked.
```
void dosomething(){
  print('ha!');
}

int assignval() {
  return 3;
}

void main() {
  final int a;
  dosomething();
  a = assignval();
  print ('$a');
}
```

Reply:
On my second thought, can this be inlined? because the `switch`-`case` below does not affect the value of `alignment`.

```
final alignment = _getAlignmentForAxisDirection(_lastOverscrollNotification?.overscroll ?? 0.0);
```